editor/configs/peopleColumns.tsx

```diff
@@ -33,7 +33,7 @@ export const ALL_PEOPLE_COLUMNS = {
       labelColumnName: "label",
     },
     valueRenderer: (record) => {
-      const value = record.injry_sev.label;
+      const value = record.injry_sev?.label;
```

Comment:
this is random but fixed a bug where if you edit the injury severity to be null be saving the Select... option, the crash details page for that crash breaks. u can see this in staging